### PR TITLE
Zoho l41 t950 6 other inconsistencies

### PR DESCRIPTION
**Pull request by thedirtypanda**
General design inconsistencies from the (internal) sheet.

### CODE REVIEW

File: application/controllers/UserManagementController.php [hunk 1]
```diff
@@ -343,7 +343,7 @@ public function actionDeleteConfirm()
     /**
      * Show some user detail and statistics
      *
-     * @param $userid int
+     * @param int $userid int
```
Comment (olleharstedt) [inline application/controllers/UserManagementController.php:346]: int x 2?

File: application/controllers/admin/quotas.php [hunk 2]
```diff
@@ -126,9 +126,13 @@ public function massiveAction()
         }
     }
 
+    /**
+     * Index
+     * @param $iSurveyId
```
Comment (olleharstedt) [inline application/controllers/admin/quotas.php:131]: Missing `int`

File: application/controllers/admin/quotas.php [hunk 3]
```diff
@@ -258,7 +265,10 @@ public function index($iSurveyId, $quickreport = false)
         }
     }
 
-
+    /**
+     * Insert Quota answer
+     * @param $iSurveyId
```
Comment (olleharstedt) [inline application/controllers/admin/quotas.php:270]: Missing `int`

File: application/controllers/admin/quotas.php [hunk 4]
```diff
@@ -287,6 +297,10 @@ public function insertquotaanswer($iSurveyId)
         }
     }
 
+    /**
+     * Delete answers 
+     * @param $iSurveyId
```
Comment (olleharstedt) [inline application/controllers/admin/quotas.php:302]: And here and below :)

File: application/controllers/UserManagementController.php (hunk 1 above)
Comment (thedirtypanda) [reply application/controllers/UserManagementController.php]: Done.

File: application/controllers/admin/quotas.php (hunk 2 above)
Comment (thedirtypanda) [reply application/controllers/admin/quotas.php:131]: Fixed.

File: application/controllers/admin/quotas.php (hunk 4 above)
Comment (thedirtypanda) [reply application/controllers/admin/quotas.php:302]: Done

**Comment by thedirtypanda** [comment]
@Shnoulle and @olleharstedt has this something to do with my changes? I never touched that file. Or could this be moved out to another PR or task?

**Comment by olleharstedt** [comment]
> @Shnoulle and @olleharstedt has this something to do with my changes? I never touched that file. Or could this be moved out to another PR or task?

Eh, no, that was for a fix I did in updatedb_helper. No idea why the comments are showing up here...

File: application/controllers/admin/quotas.php [hunk 5]
```diff
@@ -429,6 +449,10 @@ public function new_answer($iSurveyId, $sSubAction = 'new_answer')
         $this->_renderWrappedTemplate('quotas', $aViewUrls, $aData);
     }
 
+    /**
+     * New Quota
+     * @param iSurveyId
```
Comment (olleharstedt) [inline application/controllers/admin/quotas.php:454]: Missing int and $

File: application/views/admin/quotas/viewquotas_view.php [hunk 6]
```diff
@@ -78,12 +78,6 @@
                                 'header'      => gT('Completed'),
                                 'type'        => 'raw',
                                 'htmlOptions' => array('style' => 'vertical-align:top'),
-                                // 'value'=>function($oQuota)use($oSurvey){
-                                //     $completerCount =getQuotaCompletedCount($oSurvey->sid, $oQuota->id);
-                                //     $class = ($completerCount <= $oQuota->qlimit ? 'text-warning':null);
-                                //     $span = CHtml::tag('span',array('class'=>$class),$completerCount);
-                                //     return $span;
-                                // },
```
Comment (olleharstedt) [inline application/views/admin/quotas/viewquotas_view.php:86]: Why not remove this code?

**Review by olleharstedt — CHANGES_REQUESTED** [review]

File: application/views/admin/quotas/viewquotas_view.php (hunk 6 above)
Comment (olleharstedt) [reply application/views/admin/quotas/viewquotas_view.php:86]: Wait, it is removed... :facepalm:

File: application/controllers/admin/quotas.php (hunk 5 above)
Comment (thedirtypanda) [reply application/controllers/admin/quotas.php:454]: Done

File: application/controllers/UserRoleController.php [hunk 7]
```diff
@@ -38,56 +48,49 @@ public function index()
...
             ]
         );
     }
 
+
     /**
-     * Edit Role Modal.
+     * Returns the modal view for adding/editing a user role
+     *
+     * @param null $ptid
```
Comment (olleharstedt) [inline application/controllers/UserRoleController.php:71]: "null" is not a type per se. Maybe you meant "?int" or "int|null"?

Comment (thedirtypanda) [reply application/controllers/UserRoleController.php:71]: Done.

**Comment by olleharstedt** [comment]
Not sure I understand the current state of this PR. Lots of things that were already merged?

**Comment by thedirtypanda** [comment]
@olleharstedt This PR contains all changes from the UI UX Sheet. Im just waiting for the review of _Users Management: Resend login data_ (5ab4dabdfb887709b9d99b4d21d273d7e9d898ac). Its inside this branch. After that the PR is finished. One Task is left, but it will be done next sprint, because there is some specification left to be done.ß

**Comment by olleharstedt** [comment]
Hm, so how come there are so many "         $this->requirePostRequest();" changes in this PR now?

**Comment by olleharstedt** [comment]
Changed files: 86?

**Comment by thedirtypanda** [comment]
I rebased develop into my branch.

**Comment by olleharstedt** [comment]
> Rebased develop into the branch.

Hmmm.

**Comment by olleharstedt** [comment]
You might need to revert commit b2fe688b70e35b1a3ff80667c1366f20ca058d7a

**Comment by thedirtypanda** [comment]
Why? This was another PR. The base from it was this branch.

**Pull request merged**